docker-compose.yml

```diff
@@ -15,21 +15,17 @@ services:
       - ./media:/media
       - static:/static
     ports:
-      - 127.0.0.1:8890:${PORTAL_PORT}
+      - "127.0.0.1:8890:${PORTAL_PORT}"
```

Comment:
i always wondered, (why) are the quotes necessary here? i can't think of what aspect of this would break yaml parsing, but i often see this quoted.

Reply:
> i always wondered, (why) are the quotes necessary here? i can't think of what aspect of this would break yaml parsing, but i often see this quoted.

I chalk it up to bad yaml parsing, but also yaml does apparently support a lot of syntax I've never seen, so maybe there is something there.